messages/soql.query.md

```diff
@@ -56,6 +56,10 @@ Include deleted records. By default, deleted records are not returned.
 
 Time to wait for the command to finish, in minutes.
 
+# flags.output-file.summary
+
+File where records are written.
```

Comment:
```suggestion
File where records are written; only CSV and JSON output formats are supported. 
```

Reply:
Okay?  I got that from your PR comments.

Reply:
Also, how about we update the second existing example to show how to use this new flag?  For example:

- Read the SOQL query from a file called "query.txt" and write the CSV-formatted output to a file; the command uses the org with alias "my-scratch":

  <%= config.bin %> <%= command.id %> --file query.txt --output-file output.csv --result-format csv --target-org my-scratch

Reply:
@jshackell-sfdc done, ty!